vignettes/Demography.Rmd

```diff
@@ -94,32 +87,27 @@ dem_params <- set_demography(
 
 # Confirm that the custom demography has been set:
 dem_params$custom_demography
-
 ```
 
-## Simulations
+### Simulation
 
 Having established parameter sets with both default and custom demographic parameterisations, we can now run a simulation for each using the `run_simulation()` function. We'll also add a column to each identify the runs.
 
 ```{r}
-
 # Run the simulation with the default demographic set-up:
 exp_output <- run_simulation(sim_length, simparams)
 exp_output$run <- 'exponential'
 
 # Run the simulation for the custom demographic set-up:
 custom_output <- run_simulation(sim_length, dem_params)
 custom_output$run <- 'custom'
-
-
 ```
 
-## Visualisation
+### Visualisation
 
 Using barplots, we can visualise the effect of altering the demographic parameters by comparing the distribution of people among the age-classes we instructed the model to output on the final day of the simulation. The default demography is depicted in blue, while the custom demography is given in orange. Under the custom demography, the frequency in individuals in older age classes declines almost linearly, while in the default demography the number of individuals in each age class declines exponentially with age.
```

Comment:
In the last sentence, change "the frequency in individuals" to "the frequency of individuals"